src/operators/suffix.ts

```diff
@@ -32,7 +33,8 @@ export enum Suffixes {
 
 // squirrel the suffix values away so we can use it as a lookup later
 const SuffixValueLookup = new Map<string, string>();
-Object.values(Suffixes).forEach((item) => {
+Object.keys(Suffixes).forEach((key) => {
+  const item = Suffixes[key as keyof typeof Suffixes];
```

Comment:
Yeah, could even just be an array lookup since it's not that many keys. Then we could skip the Type augmentation. 🤔 

I was trying to touch as little as possible here 😄

Reply:
Using `Set` and creating a `SuffixKey` type.